src/eclingo/solver/tester.py

```diff
@@ -57,18 +57,26 @@ def __call__(self, candidate: Candidate) -> bool:
         candidate_neg = []
         candidate_assumptions = []
 
-        for literal in candidate[0]:
+        for literal in candidate.pos:
             assumption = (literal, True)
             candidate_assumptions.append(assumption)
             literal = literal.arguments[0]
             candidate_pos.append(literal)
 
-        for literal in candidate[1]:
+        for literal in candidate.extra_assumptions.pos:
```

Comment:
What is the reason of creating extra_assumptions? That is not something that was done before, right?

Reply:
I was thinking on a future optimization and it slip there. It does not bother, but it actually does nothing so far. Just let it there, for now. We need to star benchmarking before moving to that optimization. The way to test and optimization is by benchmarking and see that action improves time, as the last did with yale shooting.